### DISCUSSION TITLE
Screen Size

### DISCUSSION THREAD

**Discussion by bettywolfy** [nesbox/TIC-80#2064] [Ideas]
Could you make the tic80 screen 400x240?

or a ranged resolution system? something like
```lua
-- title: abacate
-- author: abacate
-- width: 400
-- max
-- height: 136
-- min

function TIC()
  -- Aquarium coming soon
end
```

(I know this would break the retro look of the tic80, but some projects need bigger screens and the tic80 is a good choice if it's on Raspberry, Nintendo-like Consoles, PC and Android)

**Comment by joshgoebel** [comment]
This isn't going to happen - as the resolution is baked into the very architecture of the entire system. There are other choices (and entire frameworks) if you want larger resolutions.
